.github/workflows/benchmarking.yml

```diff
@@ -120,63 +135,63 @@ jobs:
-                      { "platform": "neonvm-captest-new",      "db_size": "50gb" }]
-        }'
-
-        if [ "$(date +%A)" = "Saturday" ]; then
-          matrix=$(echo "$matrix" | jq '.include += [{ "platform": "rds-postgres", "db_size": "10gb"},
-                                                   { "platform": "rds-aurora",   "db_size": "50gb"}]')
-        fi
-
-        echo "matrix=$(echo "$matrix" | jq --compact-output '.')" >> $GITHUB_OUTPUT
-
-    - name: Generate matrix for OLAP benchmarks
-      id: olap-compare-matrix
-      run: |
-        matrix='{
-          "platform": [
-            "neon-captest-reuse"
-          ]
-        }'
-
-        if [ "$(date +%A)" = "Saturday" ]; then
-          matrix=$(echo "$matrix" | jq '.include += [{ "platform": "rds-postgres" },
-                                                   { "platform": "rds-aurora"   }]')
-        fi
-
-        echo "matrix=$(echo "$matrix" | jq --compact-output '.')" >> $GITHUB_OUTPUT
-
-    - name: Generate matrix for TPC-H benchmarks
-      id: tpch-compare-matrix
-      run: |
-        matrix='{
-          "platform": [
-            "neon-captest-reuse"
-          ],
-          "scale": [
-            "10"
-          ]
-        }'
-
-        if [ "$(date +%A)" = "Saturday" ]; then
-          matrix=$(echo "$matrix" | jq '.include += [{ "platform": "rds-postgres", "scale": "10" },
-                                                   { "platform": "rds-aurora",   "scale": "10" }]')
-        fi
-
-        echo "matrix=$(echo "$matrix" | jq --compact-output '.')" >> $GITHUB_OUTPUT
+      - name: Generate matrix for pgbench benchmark
+        id: pgbench-compare-matrix
+        run: |
+          matrix='{
+            "platform": [
+              "neon-captest-new",
+              "neon-captest-reuse",
+              "neonvm-captest-new"
+            ],
+            "db_size": [ "10gb" ],
+            "include": [{ "platform": "neon-captest-freetier",   "db_size": "3gb"  },
+                        { "platform": "neon-captest-new",        "db_size": "50gb" },
+                        { "platform": "neonvm-captest-freetier", "db_size": "3gb"  },
+                        { "platform": "neonvm-captest-new",      "db_size": "50gb" }]
+          }'
+
+          if [ "$(date +%A)" = "Saturday" ]; then
+            matrix=$(echo "$matrix" | jq '.include += [{ "platform": "rds-postgres", "db_size": "10gb"},
+                                                     { "platform": "rds-aurora",   "db_size": "50gb"}]')
+          fi
+
+          echo "matrix=$(echo "$matrix" | jq --compact-output '.')" >> $GITHUB_OUTPUT
+
+      - name: Generate matrix for OLAP benchmarks
+        id: olap-compare-matrix
+        run: |
+          matrix='{
+            "platform": [
+              "neon-captest-reuse"
+            ]
+          }'
+
+          if [ "$(date +%A)" = "Saturday" ] || ${{ github.event.inputs.run_AWS_RDS_AND_AURORA }}; then
```

Comment:
For scheduled workflow runs template `${{ github.event.inputs.run_AWS_RDS_AND_AURORA }}` is evaluated as a blank space, and this makes a bash script invalid. 
Inputs variables are evaluated only for runs that are triggered by `workflow_dispatch` event (and their default values as well, i.e. for scheduled runs you can't rely on it):

```bash
$ if [ "$(date +%A)" = "Saturday" ] || ; then echo true; else echo false; fi
bash: syntax error near unexpected token `;'
```

I would recommend use an env var to workaround this, something like:
```yaml
env:
  RUN_AWS_RDS_AND_AURORA: ${{ github.event.inputs.run_AWS_RDS_AND_AURORA || 'false' }}
```

And then use `${RUN_AWS_RDS_AND_AURORA}` in the bash part

Reply:
Good catch.